### ISSUE TITLE
Certain Address not responding correctly to any attempt of any action 

### ISSUE THREAD

**Issue by DigitalSharkLtd** [DogepartyXDP/dogewallet-desktop#22]
https://dogeparty.tokenscan.io/address/DLYAwLLB8QCLjkGRgG3YfEfnN6rP9REpJ4
Cant make dispenser
Cant send any token 
![image](https://github.com/user-attachments/assets/8be04f44-7fc1-44b3-a598-cf8923b472e9)
![image](https://github.com/user-attachments/assets/3b7f6966-046d-4991-9e63-e0ca32cda5dc)
![image](https://github.com/user-attachments/assets/1efd44f2-1c6c-4760-9135-9588aeea96a6)
![image](https://github.com/user-attachments/assets/6bd7c896-cce8-4972-9553-be49dc8c84a6)

**Comment by DigitalSharkLtd** [comment]
![image](https://github.com/user-attachments/assets/b56437ce-4772-4a4a-a5b8-ffcf036bb9bf)
![image](https://github.com/user-attachments/assets/7b14f437-8049-48ca-b205-06edfbb256a6)

**Comment by DigitalSharkLtd** [comment]
![image](https://github.com/user-attachments/assets/1d941cc8-6d24-4308-a2e5-825aa89d41c2)

**Comment by jdogresorg** [comment]
in this case it looks like it is an issue getting your private key to sign the transaction.

Can you please logout of your wallet entirely (click the power button in upper right corner) and then close the app (make sure you have your 12-word passphrase written down first).... Then open the walllet again and re-import your 12-word passphrase, this should re-create your wallet and hopefully give you back access to that address.

Not being able to access your private key for the address to sign could be related to a corrupted address index derivation path... resetting the wallet should fix it.

**Comment by DigitalSharkLtd** [comment]
I cant open/view private key for that addy looks like i have same issue that i had at the very start when i lost coins, if i relog into my wallet with 12 words i will loose this addy from a list and will never get access to it... hopefully its from my other wallet=)
but i dont have actual access to it anyway ( i mean i cant send funds or make dispenser so ill reset my wallet)
![image](https://github.com/user-attachments/assets/3dcdc71d-82f5-4aa6-a4f9-79273b9be9cb)

**Comment by DigitalSharkLtd** [comment]
yeah it may leave old addresses from a previous wallet logins ( if you just downloaded/started or logged with another phrase key before ) and replaces #1 addy in the new wallet with the previous #1 addy
